config/crd/bases/kaito.sh_workspaces.yaml

```diff
@@ -801,4 +829,7 @@ spec:
     served: true
     storage: true
     subresources:
+      scale:
+        specReplicasPath: .spec.inference.replicas
+        statusReplicasPath: .status.inference.replicas
```

Comment:
Do we need to include `labelSelectorPath`?

<img width="1259" height="182" alt="Image" src="https://github.com/user-attachments/assets/b7f8fc89-13ae-4397-bd6f-c3eb5e46fea6" />

https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#scale-subresource

Reply:
selector field has been added for scale api.